Bookish/Bookish.Web/Models/CatalogueViewModel.cs

```diff
@@ -10,11 +10,23 @@ public class CatalogueViewModel
     {
         public IEnumerable<Book> Books { get; }
         public string SearchTerm { get; }
+        public int PageNumber { get; }
 
-        public CatalogueViewModel(IEnumerable<Book> books, string searchTerm)
+        public CatalogueViewModel(IEnumerable<Book> books, string searchTerm, int pageNumber)
         {
             Books = books;
             SearchTerm = searchTerm;
+            PageNumber = pageNumber;
+        }
+
+        public string GetDisabledClass(bool isNext)
```

Comment:
I would have two methods for this instead of one method with a boolean parameter. It's best to avoid using boolean parameters like this because it makes the logic harder to follow 🙂